### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by MaslovaEV**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,12 +86,15 @@ So, what is supported:
 **Pulse plugin**
 
 Pulse Plugin is designed for visualization of virtual machines performance statistics. Currently, this CloudStack-UI extension is only compatible with ACS clusters that use the KVM hypervisor. With help of sensors that collect virtual machines performance statistics via the Libvirt API and store them in an InfluxDB datastore and RESTful statistics server, CloudStack-UI is able to display CPU, RAM, disk IO and network traffic utilization in the form of convenient visual charts.
+
 Pulse allows users of Apache CloudStack to monitor current and previous operational states of virtual machines. The plugin supports various view scales like minutes, hours, days and enables data overlays to monitor peak and average values.
+
 We consider this plugin very important for the CloudStack ecosystem as currently there is no built-in functionality to track VM operational states, although it is vital for system administrators to successfully operate virtual servers. Read more about Plugin deployment [here](https://github.com/bwsw/cloudstack-ui/wiki/Pulse-Plugin-Deployment).
```
Comment (ershovaelena) [inline README.md]: The link is wrong

File: README.md [hunk 2]
```diff
@@ -226,27 +259,40 @@ docker run -d -p 80:80 --name cloudstack-ui \
...
 
 Some operations implemented in the UI require "delayed" activities, so we use additional cleaner container that cleans objects marked for the removal.
 
 Download and start [bwsw/cloudstack-ui-cleaner](https://hub.docker.com/r/bwsw/cloudstack-ui-cleaner/) container.
 
+## Versioning
+
+```
++- Major version has breaking changes.
+| +- Minor version is synchronize with CloudStack version (major and minor without a dot).
```
Comment (ershovaelena) [inline README.md]: Check english please

File: README.md [hunk 3]
```diff
@@ -226,27 +259,40 @@ docker run -d -p 80:80 --name cloudstack-ui \
...
+| | +- Patch version has features and bug fixes.
```
Comment (ershovaelena) [inline README.md]: This nub,er is compatible with our iterations

File: README.md [hunk 4]
```diff
@@ -149,44 +157,58 @@ Our team has made a big contribution to the improvement of UX when creating a vi
...
 
+#### Snapshots
+
+We have added a section - Snapshots - that makes it easier for a user to manage snapshots created for volumes in the system. A user can view the list of snapshots and filter or group it by type or account (available to Administrators). We also added filtering by date.
+
+In the details sidebar a user can see the snapshot general information and the volume that the snapshot is created for.
+
+Here the action box also allows a user to create template or volume from the snapshot, revert a volume to the snapshot, or delete a snapshot.
+
+<a href="https://raw.githubusercontent.com/bwsw/cloudstack-ui/master/screens/snapshotsView.png" target="_blank">![Snapshots view](./screens/snapshotsView_mini.png)</a>&nbsp;&nbsp;
```
Comment (ershovaelena) [inline README.md]: Please check if pictures are displayed correctly. I do not see them

File: README.md [hunk 5]
```diff
@@ -195,17 +217,28 @@ Here domain administrators can manage existing accounts, create new accounts and
...
-2. Clone the CSUI project from GitHub.
-3. Run "npm install" command. This command installs all dependencies, which are used in the project. Also, you may use "yarn" command.
-4. Add your own proxy-conf.js file in the project folder and set the API endpoint in this file. See [proxy-conf-example](https://github.com/bwsw/cloudstack-ui/blob/master/proxy-conf-example.js).
+### Prerequisites
+
+Before you start, please, prepare Node development environment. Install Node.js or update your current node.js to latest stable version.
+
+### Download and install dependencies
+
+1. Clone the CSUI project from GitHub.
```
Comment (ershovaelena) [inline README.md]: CS-UI

File: README.md (hunk 4 above)
Comment (MaslovaEV) [reply README.md]: Please, check the text only, screenshots will be added later.

**Pull request merged**